tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "target": "es6",
+    "module": "esNext",
+    "outDir": "./dist",
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "resolveJsonModule": true,
+    "strictFunctionTypes": true,
+    "strictNullChecks": true,
+    "strictBindCallApply": true,
+    "esModuleInterop": true,
+    "experimentalDecorators": true,
+    "jsx": "react-native",
+    "jsxFactory": "React.createElement",
+    "baseUrl": "./src",
+    "paths": {
+      "@core/*": ["app/*"],
+      "@modules/*": ["modules/*"],
+      "@assets/*": ["assets/*"],
+      "@styles/*": ["app/styles/*"]
```

Comment:
do we have all aliases there? please make sure that this file, webpack config, jest config and eslint config includes the same paths' aliases

Reply:
Aliases not working from tsconfig in react native :) they need to be in babel config :)

Reply:
Should be removed form there to not confuse.